prow/plugins/label/label.go

```diff
@@ -213,6 +223,39 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 		}
 	}
 
+	mID, err := strconv.Atoi(maintainersID)
+	if err != nil {
+		log.WithError(err).Errorf("Couldn't convert the maintainers team ID parameter to int %v", maintainersID)
+	}
+	milestoneMaintainers, err := gc.ListTeamMembers(mID)
+	maintainersMap := map[string]bool{}
```

Comment:
Switch the order of this and the previous line so that the `*, err := *\n if err == nil {` stuff is adjacent..